examples/ExamplePdf.md

```diff
@@ -0,0 +1,191 @@
+# Mega Überschrift 
```

Comment:
Again, refrain from adding this file into the PR

Reply:
Noted I will remove it